cellprofiler/modules/correctilluminationcalculate.py

```diff
@@ -206,209 +226,222 @@ def create_settings(self):
-            spline parameters are not calculated automatically)</i><br>
-            This is the number of control points for the spline.
-            A value of 5 results in a 5x5 grid of splines across the image and
-            is the value suggested by the method's authors. A lower value
-            will give you a more stable background while a higher one will
-            fit variations in the background more closely and take more time
-            to compute.""" % globals())
+            "Number of spline points", 5, 4, doc="""\ 
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method and
+spline parameters are not calculated automatically)*
+This is the number of control points for the spline. A value of 5
+results in a 5x5 grid of splines across the image and is the value
+suggested by the method’s authors. A lower value will give you a more
+stable background while a higher one will fit variations in the
+background more closely and take more time to compute.
+""" % globals())
 
         self.spline_rescale = cps.Float(
-                "Image resampling factor", 2, minval=1, doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method and
-            spline parameters are not calculated automatically)</i><br>
-            This setting controls how the image is resampled to
-            make a smaller image. Resampling will speed up processing,
-            but may degrade performance if the resampling factor is larger
-            than the diameter of foreground objects. The image will
-            be downsampled by the factor you enter. For instance, a 500x600
-            image will be downsampled into a 250x300 image if a factor of 2
-            is entered.""" % globals())
+            "Image resampling factor", 2, minval=1, doc="""    
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method and
+spline parameters are not calculated automatically)*
+This setting controls how the image is resampled to make a smaller
+image. Resampling will speed up processing, but may degrade performance
+if the resampling factor is larger than the diameter of foreground
+objects. The image will be downsampled by the factor you enter. For
+instance, a 500x600 image will be downsampled into a 250x300 image if a
+factor of 2 is entered.
+""" % globals())
 
         self.spline_maximum_iterations = cps.Integer(
-                "Maximum number of iterations", 40, minval=1, doc="""
-            <i>(Used only if %(SM_SPLINES)s are selected for the smoothing method and
-            spline parameters are not calculated automatically)</i><br>
-            This setting determines the maximum number of iterations
-            of the algorithm to be performed. The algorithm will perform
-            fewer iterations if it converges.""" % globals())
+            "Maximum number of iterations", 40, minval=1, doc="""
+*(Used only if %(SM_SPLINES)s are selected for the smoothing method and
```

Comment:
Add a newline between `*(Used only if...)*` and the next line. This will insert a line break in the rendered documentation. Otherwise, the text appears on the same line.